### DISCUSSION TITLE
ANR

### DISCUSSION THREAD

**Discussion by zLShy** [Tencent/libpag#2239] [Q&A] [answered]
E  PAGAnimator.play() The scale of animator duration is turned off!
 system_server                        E  film:can't find the right tp NO.
system_server                        E  film: not init yet
system_server                        E  ANR in com.xxx.xxx(com.xxx.xxx/com.google.android.apps.chrome.Main)

**Comment by kevingpqi123** [comment, marked as answer]
这里的 log 很清晰哈：
E PAGAnimator.play() The scale of animator duration is turned off!

你搜索下 The scale of animator duration 指的是啥，在 Android 平台，开发者选项中有一个配置是 动画时长伸缩， 如果关闭，系统的 ValueAnimator 就会收到影响，全局的动画都会消失，至于 ANR 你们就分析下你们是否兼容处理了这种情况
